gammapy/background/cube.py

```diff
@@ -809,6 +809,38 @@ def plot_spectrum(self, coord, ax=None, style_kwargs=None):
             plt.close(fig)
         return ax
 
+    @property
+    def integral(self):
+        """Integral of the cube (`~astropy.units.Quantity`)"""
```

Comment:
It depends on the contents of the cube. If it is counts, it is a scalar, and it gives the total number of counts, but if it is the bg rate it will return the number of counts corrected by the livetime (units of 1/s).